packages/config/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## Unreleased
+
```

Comment:
Should include a `1.0.0-alpha.0` entry as well - (or is this taken care of automatically by changeset?)

Reply:
Not sure on the new process...cc @justinbarry